packages/react-sdk/src/core/components/ParticipantView/DefaultParticipantViewUI.tsx

```diff
@@ -93,8 +88,7 @@ export const ParticipantDetails = ({
   const call = useCall()!;
 
   const connectionQualityAsString =
-    !!connectionQuality &&
-    String(SfuModels.ConnectionQuality[connectionQuality]).toLowerCase();
+    SfuModels.ConnectionQuality[connectionQuality].toLowerCase();
```

Comment:
`!!connectionQuality` is here because at some point it might be `UNSPECIFIED`. How would this look like on the UI now?

Reply:
Missed that one, good catch. 👍